souto-app/src/components/pages/loginPage.js

```diff
@@ -214,16 +214,19 @@ export default class LoginPage extends Component {
                 spacing={3}
               >
                 <Grid item xs={12}>
-                  <Typography variant={'h6'} style={styles.text}>
+                  <Typography variant={"h6"} style={styles.text}>
                     What is your board name?
                   </Typography>
                 </Grid>
                 <Grid item xs={12}>
                   <TextField
+                    inputProps={{
+                      maxLength: 20,
```

Comment:
20 is pretty small no?
"New employee at facebook" is 21 chars. I think 40 is enough